docs/DeveloperGuide.md

```diff
@@ -579,4 +610,43 @@ For example, the user is allowed to enter these 2 commands and create 2 Persons
 * `add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 evt/1 evt/2` to add a person called **John Doe**.
 * `add n/John doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 evt/1 evt/2` to add a person called **John doe**.
 
-For future enhancement, Person names with the same sequence of characters but different casing will be considered the same Person.
+For future enhancements, Person names with the same sequence of characters but different casing will be considered the same Person.
+
+### Person name can be more flexible
+
+The current version of this application is not able to accept names that contain non-alphanumeric characters. The non-alphanumeric characters found in most names are `/` and `'`.
+For example, the user is not allowed to enter these 2 commands and create the following Persons in the list of Contacts due to the current constraint:
+* `add n/John O'neal p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 evt/1 evt/2` to add a person called **John Doe**.
+* `add n/John s/o Raju p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 evt/1 evt/2` to add a person called **John doe**.
+
+For future enhancements, Person names should be more flexible in catering for names that may contain non-alphanumeric characters in the English language.
+
+### Events that have ended can be displayed differently
+
+In this version of the application, the displayed events does not have any special indicator to show that it has "expired" or ended.
+For example, if today's date is 9th April 2023 and there are events that have end dates earlier than 9th April, they should be shown as greyed out events to indicate to the user that the event has ended.
+This Ui enhancement would help allow the users to focus on upcoming events.
+
+For future enhancements, the Ui should display past events with a special indicator or color to differentiate itself from future events.
+
+### Ability to archive events
```

Comment:
I think the ability to archive, set reminders and mark oic should be under proposed features instead of proposed enhancement because enhancements are for the bugs I think

Reply:
Oh and for ability to set reminders, can change it to ability to set and receive remidners

Reply:
Changed thank you :)